types/known/anypb/any.pb.go

```diff
@@ -284,8 +284,8 @@ func UnmarshalTo(src *Any, dst proto.Message, opts proto.UnmarshalOptions) error
 		return protoimpl.X.NewError("invalid nil source message")
 	}
 	if !src.MessageIs(dst) {
-		got := dst.ProtoReflect().Descriptor().FullName()
-		want := src.MessageName()
+		want := dst.ProtoReflect().Descriptor().FullName()
+		got := src.MessageName()
 		return protoimpl.X.NewError("mismatched message type: got %q, want %q", got, want)
```

Comment:
🤔 If you read the godoc above `// It reports an error if dst is not of the right message type.` So, `src` is properly the `want` and `dst` is properly the `got`.

However, I will admit that this is super ambiguous as to which one is supposed to be which. The text could maybe be changed to `src %q, dst %q`. That way this ambiguity of what should be `got` vs `want` can be avoided.